src/strings/PowerFxResources.en-US.resx

```diff
@@ -4579,4 +4607,32 @@
     <value>Delegation warning. The result of this argument '{0}' may be truncated for large data sets before being passed to the '{1}' function.</value>
     <comment>Error message when an argument to non-delegable function has possible delegation and resulting rows may be truncated</comment>
   </data>
+  <data name="ErrNeedPrimitive" xml:space="preserve">
+    <value>Expected a text, numeric, boolean, or date/time value.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrSummarizeNoGroupBy" xml:space="preserve">
+    <value>Summarize requires at least one grouping column.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrSummarizeInvalidArg" xml:space="preserve">
+    <value>Summarize arguments must be either existing column names to group by or formulas named with the As keyword.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrSummarizeDatasourceAsNode" xml:space="preserve">
+    <value>Summarize data source doesn't support As keyword.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrSummarizeThisGroupColumnName" xml:space="preserve">
+    <value>Column 'ThisGroup' conflicts with a reserved name. Use 'RenameColumns' function to pick a different name.</value>
```

Comment:
>RenameColumns [](http://example.com/codeflow?start=67&length=13)

In this error message the function name 'RenameColumns' and keyword 'ThisGroup' are within single quotes; the previous ones are not. We should be consistent.